pkg/ansible/run.go

```diff
@@ -160,12 +160,25 @@ func Run(flags *aoflags.AnsibleOperatorFlags) error {
 	}
 	// Create Service object to expose the metrics port(s).
 	// TODO: probably should expose the port as an environment variable
-	_, err = metrics.CreateMetricsService(context.TODO(), cfg, servicePorts)
+	service, err := metrics.CreateMetricsService(context.TODO(), cfg, servicePorts)
 	if err != nil {
 		log.Error(err, "Exposing metrics port failed.")
 		return err
 	}
 
+	// CreateServiceMonitors will automatically create the prometheus-operator ServiceMonitor resources
+	// necessary to configure Prometheus to scrape metrics from this operator.
+	services := []*v1.Service{service}
+	_, err = metrics.CreateServiceMonitors(cfg, namespace, services)
+	if err != nil {
+		log.Info("Could not create ServiceMonitor object", "error", err.Error())
```

Comment:
Looks like CI is unhappy because the error is showing up in the logs (https://travis-ci.org/operator-framework/operator-sdk/jobs/635443111#L1665-L1667), we might need to either change that log or change the way we search for errors in the test.

Maybe we could call the `error` field something like `reason` instead?

Also minor not, I don't think we need to log twice here when the service monitor is not present, maybe make these two logs into an `if .. else` like 

```
if err == metrics.ServiceMonitorNotPresent {
    // log about installing the operator
} else {
    // generic error log
}
```

just to cut down on the noise in the logs

Reply:
I changed the `error` to `reason`. However, I elected not to prevent logging 2x for two reasons:

1. Match the scaffolding provided for go based operators
1. This only happens once on startup which makes me believe this isn't an unreasonable contribution to noise.

If you would still like me to restructure the logging here @fabianvf , I can do that.

Reply:
I agree with @djzager 
I think we should keep the same impl done in go and then, since it was updated to it lastest version, I do not think that the errors will appear too. WDYT @fabianvf

Reply:
@djzager we need test locally as well operator-sdk run --local. I will do fully test with and let you know.